src/DevBetterWeb.UploaderApp/UploaderService.cs

```diff
@@ -236,10 +236,17 @@ private async Task<AnimatedThumbnailsResponse> CreateAnimatedThumbnails(long vid
     _logger.LogInformation($"Creating Animated Thumbnails");
 
     Video video = new Video();
-    while (video == null || video.Status != "available")
+    while (video.Status != "available")
     {
       Thread.Sleep(20 * 1000);
```

Comment:
Would be great if we could avoid sleeping for 20 seconds.

Reply:
from the work with Vimeo the sleep is very important maybe later we can recheck that